src/wp-includes/cache-compat.php

```diff
@@ -141,3 +141,21 @@ function wp_cache_flush_runtime() {
 		return wp_using_ext_object_cache() ? false : wp_cache_flush();
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_flush_group' ) ) :
+	/**
+	 * Removes all cache items in a group.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @see WP_Object_Cache::flush_group()
+	 * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+	 *
+	 * @param string|array $group name(s) of group to remove from cache.
+	 *
+	 * @return bool|Array True or array of bool in array passed on success, false on failure group not found.
+	 */
+	function wp_cache_flush_group( $group ) {
+		return wp_cache_flush();
```

Comment:
Maybe this should have a doing it wrong or something here. Tell developers to no call this function unless your objet cache supports it.

Reply:
added